### PR TITLE
added house sql files

### PR DESCRIPTION
**Pull request by XeAri**

### CODE REVIEW

File: src/servers/sapphire_zone/Zone/House.h [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#ifndef SAPPHIRE_HOUSE_H
+#define SAPPHIRE_HOUSE_H
+
+#include "Forwards.h"
+#include "HousingZone.h"
```
Comment (SapphireMordred) [inline src/servers/sapphire_zone/Zone/House.h]: no need to include that here

File: src/servers/sapphire_zone/Zone/Land.h [hunk 2]
```diff
@@ -99,6 +101,13 @@ namespace Core
     char m_ownerName[31];
     char fcTag[7];
     uint8_t m_tag[3];
+
+    //house
+    uint32_t m_houseId;
+    HousePtr m_house;
```
Comment (SapphireMordred) [inline src/servers/sapphire_zone/Zone/Land.h]: i'd name it m_pHouse to stay consistent with other ptr members we use

File: src/servers/sapphire_zone/Zone/Land.h [hunk 3]
```diff
@@ -99,6 +101,13 @@ namespace Core
     char m_ownerName[31];
     char fcTag[7];
```
Comment (SapphireMordred) [inline src/servers/sapphire_zone/Zone/Land.h]: no m_?

File: src/servers/sapphire_zone/Zone/Land.h [hunk 4]
```diff
@@ -99,12 +99,12 @@ namespace Core
...
+    char m_fcTag[7];
     uint8_t m_tag[3];
 
     //house
     uint32_t m_houseId;
-    HousePtr m_house;
+    HousePtr m_pHouse;
 
     //manager
     HousingMgrPtr pHousingMgr;
```
Comment (SapphireMordred) [inline src/servers/sapphire_zone/Zone/Land.h]: same thing here. missing m_

File: src/servers/sapphire_zone/Zone/Land.cpp [hunk 5]
```diff
@@ -100,7 +100,7 @@ uint32_t Core::Land::getCurrentPrice() const
 void Core::Land::setPreset( uint32_t id )
 {
   auto pExdData = g_fw.get< Data::ExdDataGenerated >();
-  auto info = pExdData->get< Core::Data::HousingPreset >( pHousingMgr->convertItemId( id ) );
+  auto info = pExdData->get< Core::Data::HousingPreset >( getHousingMgr()->convertItemId( id ) );
```
Comment (SapphireMordred) [inline src/servers/sapphire_zone/Zone/Land.cpp]: nullptr check to be save here.

File: sql/house.sql [hunk 6]
```diff
@@ -0,0 +1,15 @@
+CREATE TABLE `house` (
+	`LandSetId` INT UNSIGNED NULL,
+	`HouseId` INT UNSIGNED NOT NULL AUTO_INCREMENT,
+	`BuildTime` BIGINT NULL,
+	`Aetheryte` SMALLINT NULL, 
+	`Welcome` SMALLINT NULL,  
+	`Comment` binary(193) DEFAULT "", 
+	`Like` BIGINT NULL,
```
Comment (takhlaq) [inline sql/house.sql]: keyword in sql, might need to quote in backticks when querying (or maybe rename to Likes/Endorsements?)

File: sql/housepartsitem.sql [hunk 7]
```diff
@@ -0,0 +1,9 @@
+CREATE TABLE `housepartsitem` (
+	`HouseId` INT UNSIGNED NULL,
+	`itemId` INT UNSIGNED NULL,
```
Comment (takhlaq) [inline sql/housepartsitem.sql]: casing is different to usual

File: sql/housepartsitem.sql [hunk 8]
```diff
@@ -0,0 +1,9 @@
...
+	`CharacterId` INT UNSIGNED NULL,
+	`color` INT UNSIGNED DEFAULT '0',
```
Comment (takhlaq) [inline sql/housepartsitem.sql]: casing is different to usual

File: sql/housepartsstorage.sql [hunk 9]
```diff
@@ -0,0 +1,18 @@
+CREATE TABLE `housepartsstorage` (
+	`LandSetId` INT UNSIGNED NULL,
+	`HouseId` INT UNSIGNED NULL,
+	`storageId` BIGINT,
```
Comment (takhlaq) [inline sql/housepartsstorage.sql]: casing

File: sql/housepartsstorage.sql [hunk 10]
```diff
@@ -0,0 +1,18 @@
+CREATE TABLE `housepartsstorage` (
+	`LandSetId` INT UNSIGNED NULL,
+	`HouseId` INT UNSIGNED NULL,
+	`storageId` BIGINT, 
+	`type` INT NULL,
+	`index` INT NULL,
```
Comment (takhlaq) [inline sql/housepartsstorage.sql]: type and index will need to be queue quoted with \` or these names can be changed to PartType PartIndex etc

File: sql/housepartsstorage.sql [hunk 11]
```diff
@@ -0,0 +1,18 @@
...
+	`container_0` INT UNSIGNED DEFAULT '0',
+	`container_1` INT UNSIGNED DEFAULT '0',
+	`container_2` INT UNSIGNED DEFAULT '0',
+	`container_3` INT UNSIGNED DEFAULT '0',
+	`container_4` INT UNSIGNED DEFAULT '0',
+	`container_5` INT UNSIGNED DEFAULT '0',
+	`container_6` INT UNSIGNED DEFAULT '0',
+	`container_7` INT UNSIGNED DEFAULT '0',
```
Comment (takhlaq) [inline sql/housepartsstorage.sql]: casing

**Pull request closed without merging**